### PR TITLE
nlb and nba data

### PR DESCRIPTION
**Pull request by Dz373**

### CODE REVIEW

**Comment by Dz373** [comment]
data for basketball, baseball, soccer, and hockey

File: back-end/index.js [hunk 1]
```diff
@@ -16,7 +16,10 @@ app.use('/api/auth', authRoutes);
 
 // protected route
 app.use('/api/games', authMiddleware, gameRoutes)
-app
+
+//test route
+app.use('/api', gameRoutes)
```
Comment (vphatfla) [inline back-end/index.js]: what is this? @Dz373

**Review by vphatfla — CHANGES_REQUESTED** [review]

File: back-end/index.js (hunk 1 above)
Comment (Dz373) [reply back-end/index.js]: app.use('/api/games', authMiddleware, gameRoutes) required authorization

Comment (vphatfla) [reply back-end/index.js]: yes, and that's how it's supposed to be, you don't want your APIs to be public

Comment (Dz373) [reply back-end/index.js]: I was using it to test that the APIs worked. I was going to get rid of it after. @vphatfla

Comment (vphatfla) [reply back-end/index.js]: please remove it now, since you are merging to main branch

**Review by vphatfla — APPROVED** [review]

**Pull request merged**